modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -20,10 +19,25 @@ def index
           render json: claims
         end
 
+        def show
+          bgs_claim = bgs_service.benefit_claims.find_claim_details_by_claim_id(claim_id: params[:id]) || {}
+          claim_details = bgs_claim.dig(:bnft_claim_detail)
+
+          # TODO: do we need to compare BGS vs Lighthouse claims like in claims#index?
+
+          if claim_details.present?
+            claim = { id: claim_details[:bnft_claim_id], type: claim_details[:bnft_claim_type_nm] }
+            render json: claim
+          else
+            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
+          end
```

Comment:
can we switch this up to use (the thing i can never remember the name of 😞), like this:
```
raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if claim_details.blank?

render json: { id: claim_details[:bnft_claim_id], type: claim_details[:bnft_claim_type_nm] }
```
i just think this structure always reads a bit better than nested if/else statements like this

Reply:
> "_the thing i can never remember the name of_"

Guard clause?

Reply:
I updated this aspect of the code. the logic here got refactored, but there's a different `if/else` due to the potential lookup in BGS, if needed